src/main/resources/application-docker-local.properties

```diff
@@ -21,15 +21,13 @@ spring.datasource.password = root
 spring.datasource.testWhileIdle = true
 spring.datasource.validationQuery = SELECT 1
 
-cart.service.url = http://localhost:8083
-product.service.url = http://localhost:8081
 user.service.url = http://localhost:8080
 
 # Email configurations
 spring.mail.host = smtp.hostinger.com
 spring.mail.port = 587
 spring.mail.username = noreply@springwebstore.com
-spring.mail.password = ${HOSTINGER_EMAIL_PASS}
+spring.mail.password = Brut6vdWNgK-?nq
```

Comment:
would be nice not to have the actual password here 😸 
maybe set it as a local env variable